spec.md

```diff
@@ -2110,6 +2230,57 @@ Condition | gRPC Code | Description | Recovery Behavior
 | Call not implemented | 12 UNIMPLEMENTED | NodeGetInfo call is not implemented by the plugin or disabled in the Plugin's current mode of operation. | Caller MUST NOT retry. Caller MAY call `ControllerGetCapabilities` or `NodeGetCapabilities` to discover Plugin capabilities. |
 
 
+#### `NodeExpandVolume`
+
+A Node Plugin MUST implement this RPC call if it has `EXPAND_VOLUME` node capability.
+This RPC call allows CO to expand volume on a node.
+
+`NodeExpandVolume` ONLY supports expansion of already node published or node staged volumes on given `volume_path`.
+A plugin may return `FAILED_PRECONDITION` error if underlying volume can not be expanded when published or staged.
+
+Example:
```

Comment:
I think we can simplify the following two sections by saying:

`NodeExpandVolume` MUST be called after calling `NodeStageVolume` and is successful if the plugin has `STAGE_UNSTAGE_VOLUME` node capability.
`NodeExpandVolume` MAY be called before or after `NodePublishVolume`.

Also, this does not look like "example". So remove it.